src/OutboundServer/api.yaml

```diff
@@ -562,9 +562,7 @@ components:
           description: >
             Object representing the last error to occur during a transfer process. This may be a Mojaloop API error returned from another entity in the scheme
             or an object representing other types of error e.g. exceptions that may occur inside the scheme adapter.
-          oneOf:
-            - $ref: '#/components/schemas/transferError'
-            - $ref: '#/components/schemas/generalError'
+          $ref: '#/components/schemas/transferError'
```

Comment:
why did you make this change? I believe it is still possible for either of these to be returned?

Reply:
I haven't seen the code that could generate this general error